### PR TITLE
Update BuildTools to 1.0.26-prerelease-00815-06

### PR DESCRIPTION
**Pull request by ericstj**
/cc @weshaggard @SedarG

### CODE REVIEW

**Comment by ericstj** [comment]
@joperezr this fixes the issue I noticed in the dev-api merge.

**Comment by ericstj** [comment]
@weshaggard looks like the response file change broke this.

@JeremyKuhne if you'd like I can roll the nuget.framework workaround into this PR instead (to avoid the semaphore rename)

**Comment by JeremyKuhne** [comment]
@ericstj I'm fine with you putting that in this PR- note that I had to make changes to deal with the fact that both 0 & 1 are success codes for Robocopy. My PR is up to date.

**Comment by JeremyKuhne** [comment]
lgtm :+1:

File: init-tools.cmd [hunk 1]
```diff
@@ -60,15 +60,24 @@ if NOT exist "%BUILD_TOOLS_PATH%init-tools.cmd" (
...
 set INIT_TOOLS_ERRORLEVEL=%ERRORLEVEL%
 if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
-	echo ERROR: An error occured when trying to initialize the tools. Please check '%INIT_TOOLS_LOG%' for more details. 1>&2
-	exit /b %INIT_TOOLS_ERRORLEVEL%
+  echo ERROR: An error occured when trying to initialize the tools. Please check '%INIT_TOOLS_LOG%' for more details. 1>&2
+  exit /b %INIT_TOOLS_ERRORLEVEL%
+)
+
+echo Updating CLI NuGet Frameworks map...
+robocopy "%TOOLRUNTIME_DIR%" "%TOOLRUNTIME_DIR%\dotnetcli\sdk\%DOTNET_VERSION%" NuGet.Frameworks.dll /XO >> "%INIT_TOOLS_LOG%"
```
Comment (weshaggard) [inline init-tools.cmd]: I understand why you are doing this but I don't really like this hack. I suppose the real fix is to get an updated CLI with these updates correct? if so we should at least add a comment here and file an issue.

**Pull request merged**